src/lib/components/schedule/schedule-form-view.svelte

```diff
@@ -123,11 +122,9 @@
       <h1 class="font-base mt-8 ml-0 text-2xl">{title}</h1>
     </header>
     <form class="mb-4 flex w-full flex-col gap-4 md:w-2/3 xl:w-1/2">
-      {#if $error}
-        <Alert intent="error" title="" bold>
-          {$error}
-        </Alert>
-      {/if}
+      <Alert intent="error" title="" class={$error ? '' : 'hidden'} bold>
```

Comment:
So I went back and forth about this and experimented with adding a `hidden` prop. Definitely something we can add, but I wasn't sure if it made the most sense since it's technically something that can (and perhaps should) be controlled by it's parent or wherever it's being used.

Happy to have a further discussion on this!

Reply:
If it's a prop, the parent can still control it, right?

```
<Alert intent="error" hidden={!$error} />
```

I just don't want to have to think about which specific css class to add to ensure proper accessibility of the component.

Reply:
Yeah that's exactly how I had the prop, but something felt not right about it. I'm going to change it back and see if I still have any opinions worth investigating.

Reply:
Updated [here](https://github.com/temporalio/ui/pull/1595/commits/005b9f93140f3e975ab71cb478a4eb82f17250b9) ✅